CMakeLists.txt

```diff
@@ -58,6 +58,12 @@ IF(${CMAKE_CXX_STANDARD} LESS 14)
   MESSAGE(FATAL_ERROR "DD4hep requires at least CXX Standard 14 to compile")
 ENDIF()
 
+# ----for APPLE scripts have to set the DYLD_LIBRARY_PATH
+if(APPLE)
+  set(USE_DYLD 1)
+  set(CMAKE_MACOSX_RPATH 1)
```

Comment:
`set(CMAKE_MACOSX_RPATH 1)` is set in `cmake/DD4hepBuild.cmake` along with all the rpath treatment. And `USE_DYLD` could be drooped entirely in favor of using the variable `APPLE`

Reply:
Sure, that works. Requested changes made in commit d926c9692b5d9aacb96585ba756b7b42794c6115